test/true_one_to_many.jl

```diff
@@ -0,0 +1,50 @@
+include("../MCParticle.jl")
+
+function test_copy()
```

Comment:
I am not sure if `test_copy` is the best name in this case, as we are testing relations (in podio speak), resp. references here.